client/main.lua

```diff
@@ -171,9 +168,8 @@ end)
 RegisterNetEvent('qb-vehiclekeys:client:AddKeys', function(plate)
     KeysList[plate] = true
 
-    local ped = cache.ped
-    if IsPedInAnyVehicle(ped, false) then
-        local vehicle = GetVehiclePedIsIn(ped)
+    if IsPedInAnyVehicle(cache.ped, false) then
+        local vehicle = GetVehiclePedIsIn(cache.ped)
```

Comment:
```suggestion
        local vehicle = cache.vehicle
```
Either use cache.vehicle or remove all `vehicle` and use cache.vehicle looks better